config/custom-environment-variables.yml

```diff
@@ -14,6 +14,8 @@ public:
 
   redirectUri: 'REDIRECT_URI'
 
+  rasterUrl: 'RASTER_URL'
```

Comment:
Do you think it makes sense to configure a `RASTER_URL` in staging so that something like https://shields-staging-pr-3644.herokuapp.com/appveyor/ci/gruntjs/grunt.png works, or does it just become confusing to mix staging and production architecture?

Reply:
Yea, I was thinking it would be nice to have one for the staging environment, though perhaps it's not necessary for PRs. I _think_ we could configure the env var in the staging app and that way it'll apply only to staging (not PRs).

Reply:
its something we can deal with as an env var in heroku anyway, rather than something that needs changing in this PR